### ISSUE TITLE
Provide option to generate circular images

### ISSUE THREAD

**Issue by itaditya** [ameerthehacker/named-avatar-generator#3]

**Comment by ameerthehacker** [comment]
Yes i thought of that but most of them would like to obtain that through CSS

**Comment by itaditya** [comment]
as this plugin is server-side, many devs may use it along with an API, if that's the case you can't say that they necessarily use the API on the web. If they are using it in an Android app then I don't know if there is a technique for circular  images.

**Comment by ameerthehacker** [comment]
Yes, that is a valid point. I will add that this also to the TODO for next release

**Comment by ameerthehacker** [comment]
Whoever picking this up please refer https://www.npmjs.com/package/jimp
